vms/proposervm/proposer/windower.go

```diff
@@ -37,7 +37,7 @@ const (
 var (
 	_ Windower = (*windower)(nil)
 
-	ErrNoProposersAvailable = errors.New("no proposers available")
```

Comment:
this becomes an internal error that does not require a special error. Instead we should singla the case where no validators are available to be able to handle it appropriately